README.md

```diff
@@ -42,6 +42,7 @@ module.exports = {
   addons: [
     'storybook-addon-designs',
     // Or as a Tab instead of a Panel
+    ['storybook-addon-designs', { target: 'tab' }]
```

Comment:
I'm not sure but is this babel-style notation works? (I've edited my initial comment, sorry)

https://github.com/pocka/storybook-addon-designs/pull/47#discussion_r417115221

Reply:
I misread the initial comment. updated the readme to the right syntax